### PR TITLE
Added NavBar and created Dashboard and project routing skeleton

### PR DESCRIPTION
**Pull request by simalicrum**
### What this PR does (required):
- Added Navbar with Login/Logout and created a routing structure for the project

### Screenshots / Videos (front-end only):

image

### Any information needed to test this feature (required):
- Requires mongodb backend.

### CODE REVIEW

File: client/src/components/Navbar/LoggedOut/LoggedOut.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Box from '@material-ui/core/Box';
+import React from 'react';
+import { Link } from 'react-router-dom';
```
Comment (rajivtitus) [inline client/src/components/Navbar/LoggedOut/LoggedOut.tsx:5]: 👍🏽

File: client/src/components/Navbar/LoggedIn/LoggedIn.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  const handleLogout = () => {
+    handleClose();
+    logout();
+  };
+
+  return (
+    <React.Fragment>
+      <Box mr={8}>
+        <Link href="/dashboard/my-sitters" color="inherit">
```
Comment (rajivtitus) [inline client/src/components/Navbar/LoggedIn/LoggedIn.tsx:38]: @simalicrum You may have missed this! Please use react-router's Link and not the one from MUI :)

File: client/src/components/Navbar/LoggedIn/LoggedIn.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          <Typography variant="subtitle1">MySitters</Typography>
+        </Link>
+      </Box>
+      <Box mr={8}>
+        <Link href="/dashboard/messages" color="inherit">
```
Comment (rajivtitus) [inline client/src/components/Navbar/LoggedIn/LoggedIn.tsx:43]: Same as comment above

**Review by rajivtitus — APPROVED** [review]
Thanks for making all those changes! Just a minor comment is pending, please make the necessary change before merging :)

File: client/src/components/Navbar/LoggedIn/LoggedIn.tsx (hunk 2 above)
Comment (simalicrum) [reply client/src/components/Navbar/LoggedIn/LoggedIn.tsx:38]: Fixed

**Pull request merged**